### DISCUSSION TITLE
Orthographe

### DISCUSSION THREAD

**Discussion by Fredimimi** [L-Sherry/French-CC#17] [General]
Hello,

J'ai trouvé une petite "rareté".
Je suis [tristement stupéfait]. J'esperait vraiment que cette fois-là serait différente, mais hélas...

"J'espérais"

**Comment by L-Sherry** [comment]
C'est corrigé.

**Comment by Fredimimi** [comment]
Hello, 

J'en ai trouvé un autre : 
"Bon, pendant que tu étais occupée dehors, nous n'avons pas chaumé non plus !

chaumé -> chômé

**Reply by Fredimimi** [reply]
Et un autre, un objet : 

Brassard Etincèle

Soit "étincelant" (ou "étincelle") 
étant donné que le Brassard étincelant est l'upgrade de Brassard Brillant

---
Istatainment à perdu beaucoup de son ancien esprit d'innovation.

à -> a

---
Il y a encore quelques mots "vielle" à corriger en "vieille"

---

Donc, Pisteur. Trouve-moi un morceau d'[Très Grande Braise]. Je suis sûr...

d' -> de

**Reply by L-Sherry** [reply]
J'ai pris étincelant du coup.

Par contre je trouve plus aucune trace de "vielle", j'ai normalement tout corrigé dans b735d99d8195da2837d3bdb4b7b5e49f456d4bf8 et depuis j'ai de la conf dans config.json qui va faire que j'aurai un gros warning si il en reste.

Donc je penche plutôt pour une version de french-cc qui n'est pas à jour.

**Reply by Fredimimi** [reply]
En effet, j'ai encore la version 1.4.0 téléchargée dans les releases.

**Comment by Fredimimi** [comment]
Un petite coquille, c'est une réplique de Carla
Peu importe ! Je suis sûr que l'on finira par se comprendre....
sûr --> sûre

**Reply by L-Sherry** [reply]
... C'était loin d'être la seule erreur entre sûr et sûre.

J'avais essayé un moment de bidouiller grammalecte (le correcteur grammatical que j'utilisais) pour pouvoir forcer 'je' à être masculin ou féminin, mais j'avais pas vraiment réussi.

Sinon tu veux pas une place dans les crédits ?

**Reply by Fredimimi** [reply]
Oui, pourquoi pas :)

Trouvé une autre faute... du même style 👍 
Emilie : Quand j'étais petit, il y avait des vaches chez les voisins
petite

**Reply by Fredimimi** [reply]
Il s'agit de la traduction de "Holiday Man", traduit en homme vacance.

Il est possible de le traduire par "Personnage des fêtes".
Il y a aussi le nom du personnage qui est resté en "Holiday Man".

D'après une info, il s'agit d'un personnage tirée de One Punch Man.

**Comment by L-Sherry** [comment]
Tiens j'avais pas remarqué que le nom du personnage n'est pas traduit. Effectivement les développeurs du jeu n'ont pas mis de LangLabel sur son nom donc il y a le même problème dans les traductions officielles. Et il faut croire que la [traduction russe a patché ça](https://github.com/CCDirectLink/crosscode-ru/blob/master/enhanced-ui/src/locale-json-patches.ts)

Et sinon oui, "Fête" est probablement une meilleure traduction de Holiday, d'autant qu'en allemand ils ont traduit ça en Festtagsmann, il n'y a pas de confusion possible. Je ne sais pas si `Personnage des fêtes` ça rentre par contre.

**Reply by Fredimimi** [reply]
Si ca ne passe pas, tu peux essayer "Le gars des fêtes", avec une petite transgression dans la traduction avec l'ajout du "le"

C'est dans l'esprit du texte original

**Reply by L-Sherry** [reply]
"Personnage" ça me parait un peu curieux, surtout si c'est une référence à un super-héros. l'"Homme chauve-souris" ça sonne mieux que "Personnage chauve-souris" ou "Gars chauve-souris".

Je pensais à l'"Homme Jour Férié", ça ne devrait pas prendre beaucoup de place, et ça devrait être plus explicite.

**Reply by Fredimimi** [reply]
Difficile à dire, c'est un peu comme s'il y a ce petit coté autodérision sur la façon de se présenter.

Ou alors dire : Homme des fêtes, ou l'Homme des jours fériés, ou encore, mais là, j'explose le "format", l'homme des fêtes de fin d'année.
